ra/ra.go

```diff
@@ -1418,7 +1431,8 @@ func (ra *RegistrationAuthorityImpl) issueCertificateInner(
 		return nil, nil, wrapError(err, "parsing final certificate")
 	}
 
-	ra.countCertificateIssued(ctx, int64(acctID), parsedCertificate.DNSNames)
+	dnsNames := slices.Clone(parsedCertificate.DNSNames)
+	go ra.countCertificateIssued(ctx, int64(acctID), dnsNames, isRenewal)
```

Comment:
You could inline this call to Clone, so the `dnsNames` variable isn't hanging around for someone else to accidentally use lower down.